### ISSUE TITLE
Run 5 SL6

### ISSUE THREAD

**Issue by jenkinsSAGrid** [SouthAfricaDigitalScience/FMD-codes-from-T.-Neff-#6]
This was created via the API at the start of build 5

**Comment by brucellino** [comment]
Linker error, couldn't find LAPACK : 

```
gcc -L. -L/apprepo/generic/sl6/x86_64/lapack/3.6.0-gcc-5.2.0/lib -L/apprepo/generic/sl6/x86_64/ncurses/5.9/lib -L/apprepo/generic/sl6/x86_64/openmpi/1.8.8-gcc-5.2.0/lib -L/apprepo/generic/sl6/x86_64/gcc/5.2.0/lib64 -o gnd2fmdpara gnd2fmdpara.o git_version.o -lfmd -lnumerics -lmisc -llapack -lblas -lgfortran -lm -lz
/usr/bin/ld: cannot find -llapack
collect2: error: ld returned 1 exit status
make: *** [gnd2fmdpara] Error 1
```

the libs are in `lib64` instead of `lib`

**Comment by brucellino** [comment]
fixed in 2c1835a
